modules/shared/components/Menu/Menu.tsx

```diff
@@ -1,18 +1,20 @@
 import React from "react";
+import Link from "next/link";
+
 import {
+  Box,
   Drawer,
   DrawerBody,
-  DrawerOverlay,
-  DrawerContent,
   DrawerCloseButton,
-  useDisclosure,
-  Text,
+  DrawerContent,
+  DrawerOverlay,
   Flex,
-  Box,
+  Text,
+  useDisclosure,
 } from "@chakra-ui/react";
-import MenuLogo from "../Icons/MenuLogo";
+
 import Hamburguer from "../Icons/Hamburguer";
-import Link from "next/link";
+import MenuLogo from "../Icons/MenuLogo";
 
 const Links = [
   { title: "Colecciones", href: "/Collection" },
```

Comment:
Si podes actualizate el path a home
```suggestion
  { title: "Colecciones", href: "/" },
```